changelog/20251103_feature_mongodbsearch_mongodb_deployments_using_x509.md

```diff
@@ -0,0 +1,7 @@
+---
+kind: feature
+date: 2025-11-03
+---
+
+* **MongoDBSearch**: MongoDB deployments using X509 internal cluster authentication are now supported. Previously MongoDB Search required SCRAM authentication among members of a MongoDB replica set.
```

Comment:
I think it's worth mentioning that mongot itself still cannot authenticate using x509, so SCRAM auth must be enabled for clients to connect (but it might be both x509, but SCRAM is mandatory for search)